client/components/editor/structure/TesList.vue

```diff
@@ -72,8 +71,7 @@ export default {
       });
     },
     lastPosition() {
-      const lastItem = last(this.list);
-      return lastItem ? lastItem.position + 1 : 1;
+      return this.list.length + 1;
```

Comment:
Given 2 elements, if you sort them couple of times (move 1 after 2 and then repeat the action) lastPosition won't be the right one because last one is last.position + 1. Not length + 1.